src/imio/smartweb/policy/__init__.py

```diff
@@ -1,9 +1,60 @@
 # -*- coding: utf-8 -*-
 
 from AccessControl import allow_module, allow_class, allow_type
+from collective.pivot.config import getFamilyProperties
+from collective.pivot.utils import _
+from collective.pivot.utils import add_family
+from collective.pivot import setuphandlers
+from plone import api
 from requests.models import Response
 
 # Useful for code used in RestrictedPython (collective.themefragments)
 allow_module("requests")
 allow_class(Response)
 allow_type(type("requests.models.Response"))
+
+
+def pivot_post_install(context):
+    """Post install script"""
+    current_lang = api.portal.get_current_language()[:2]
+    site = api.portal.get()
+    if not site.get("PIVOT"):
+        folder = api.content.create(
+            type="imio.smartweb.Folder",
+            container=site,
+            title=_("PIVOT", context=site, target_language=current_lang),
+        )
+        pivot_add_default_categories(folder)
+
+
+setuphandlers.post_install = pivot_post_install
```

Comment:
If you need this line (pure Python monkey patch), it means that the `<monkey:patch` zcml directive doesn't work, right ?  If so, we should not use `collective.monkeypatcher` at all.

Otherwise, it is duplicate.